### PR TITLE
Feature/dockerize

### LINKED ISSUE #59: Dockerize
* Add a Dockerfile that runs npm as part of the docker build and runs the server as part of the docker run
* Move configuration to ENV so it can be set through docker/rancher

Stretch goal #60

### PR DESCRIPTION
**Pull request by McJeffr**
Added support for Docker. See the added README.md for more information about how the configuration part works / what options are available for configuration.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+FROM node:12.16.3
```
Comment (McJeffr) [inline Dockerfile:1]: This is the latest stable version of node.js.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+Since the provided configuration file has been fully set up to work with Shotbow's
+Discord server, you might need to manually update it. For this, you can choose from
+a variety of options.
+
+1. Update the `config/default.js` configuration file. This repository makes use of the
+[config](https://www.npmjs.com/package/config) npm-package.
+
+2. Add a `config/production.js` configuration file. The `docker build`-command sets
+the `NODE_ENV` to `production`. The [config](https://www.npmjs.com/package/config) package
+will automatically overwrite any values defined in `config/production.js` with those
```
Comment (navarr) [inline README.md]: Do you mean this the opposite way?

**Comment by navarr** [comment]
Thanks, McJeffr! This is exactly what we need!

File: README.md (hunk 2 above)
Comment (McJeffr) [reply README.md]: Correct, I fixed it.

**Review by navarr — APPROVED** [review]

**Pull request merged**